### ISSUE TITLE
HU-Transform needs to filter by HU status

### ISSUE THREAD

**Issue by metas-ts** [metasfresh/metasfresh-webui-api#606]
### Is this a bug or feature request?
Bug
### What is the current behavior?
Example:
* you have a LU with TUs and CU
* you remove all the TUs from the LU => the LU gets destroyed
  * still shown in the HU editor..maybe not nice, but not dealt with in this issue
* you pick the TU and select transform
  => the action add TU to existing LU is shown (because the destroyed LU is still counted) => NOK

#### Which are the steps to reproduce?

Play with it

### What is the expected or desired behavior?

Basically: 
* if you select an HU, you you shall only be allowed to move or add it to a destination HU that has the same status
* generally, destroyed HUs are "out"

**Comment by metas-lc** [comment]
IT
1. create new purchase order, add a line + hu, complete
2. go to receipt candidates and open hu editor with 2 LUs, 1 TU on each
3. select the TU from LU1, use transform to move it on existing LU (LU2)
=> TU is moved, LU is destroyed OK
4. select 1 TU from LU2, use transform
=> there is no option to move the TU on exsting LU OK
